README.rst

```diff
@@ -875,19 +875,59 @@ Get command details
 
     GET /api/v1/controller/device/{device_id}/command/{command_id}/
 
-Get device coordinates
+Get device location
+^^^^^^^^^^^^^^^^^^^
+
+.. code-block:: text
+
+    GET /api/v1/controller/device/{id}/location/?key={device key}
```

Comment:
The value of the name field is empty in edit page.

Reply:
Haa, Yes @atb00ker all the fields except the geometry, will be empty, because we have hindred the representation of it, which changes the hierarchy to which the values are mapped with fields. This is a problem with this module  https://github.com/openwisp/django-rest-framework-gis I have mentioned it in the summary. This can be of future work, I triedmy best to fix it, but couldn't I think i will have to understand the `django-rest-framework-gis` module to fix it. :thinking:

Reply:
For this reason I haven't used this class in locaiton endpoint serializer. so awating a reply here:- https://github.com/openwisp/openwisp-controller/pull/494#discussion_r688421609

Reply:
Also here I would not document this endpoint showing the key example.